CODEOWNERS

```diff
@@ -1 +1,2 @@
+#GUSINFO:MS CX Engineering,MS CX (DOCS)
 emileeburkeVT @mulesoft/team-docs
```

Comment:
Do you need an @ for Emilee?

Reply:
Hmmm I was thinking that too but it was always like that. I'll double check